tests/countries/test_bermuda.py

```
@@ -237,3 +237,32 @@ def test_2025(self):
             ("2025-12-25", "Christmas Day"),
             ("2025-12-26", "Boxing Day"),
         )
+
+    def test_l10n_default(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)